### ISSUE TITLE
Landing Page Improvement

### ISSUE THREAD

**Issue by pilotnbr1** [bortek/EZ-WifiBroadcast#162]
More of an observation.. Looking at the insights for this repository and traffic, most unique visitors are going to the home page. I believe that page needs to be significantly improved in order to attract users.
-move all older links to past images to the history page. Keep only most recent stable image there.
-make page more informational as to what wbc is
-perhaps some multimedia content- @htcohio seemed interested in video
-basically this is where the project will be “sold”

I can begin work work on this if there is support. I just don’t want to unilaterally change such a significant page.

![93e21f8d-53a9-453c-b540-4075e1fb2df1](https://user-images.githubusercontent.com/11220330/47089767-cc693200-d221-11e8-8a90-d2a59966d3a7.png)

**Comment by bortek** [comment]
I think the firsts page is the one that comes from the readme.md file. We have all the content on wiki. Is it possible to have a landing page a wiki main page? That way we would have one landing page instead of readme.md

**Comment by careyer** [comment]
@pilotnbr1 : This is an awesome idea! We could also use maybe khomyakk's 31km world record flight and reedit the video to add some "teaser" character to it... This way we would have both a teaser and an awesome tech demo of what this system is capable of. I know that khomyakk is willing to share the video because he sent me the full 6GB file in the past already. Of course we need to ask him again if it is okay with him...

But what is key is to have a very clean and professional looking video. Nothing quickly thrown together or showing doggy stuff with cables and hotglue hanging all over the place for example.

**Update:** We should also consider creating a dedicated YouTube channel for EZ-Wifibroadcast that is not hosted by a single individual but is rather tied and stays with the project. Login Credentials should be shared among the contributors. We could host all multimedia content there.

**Comment by careyer** [comment]
@bortek : I could not find any information on how to redirect the main landing page of a repository to its wiki.... However I have asked a question how one could achieve this here: https://github.com/contact

**Comment by careyer** [comment]
Just in case I have reserved the account for future use:
![ez-wifibroadcast_yt_channel](https://user-images.githubusercontent.com/6156251/47095461-9b8efa00-d22d-11e8-8ece-7fae34b7d6a2.JPG)

We should agree on a code of conduct using it.
- Credentials should be shared at least among two users 
- Any upload should be approved by a plenum 
- Monetization should always be truned off on all videos

**Comment by RespawnDespair** [comment]
If we were to have an official logo, we could also set it as a watermark on the channel...

**Comment by bortek** [comment]
Googled for half-an-hour and tested different suggestion on linking README.md to Wiki but it seems not possible. 

Which means that we have to create a landing page in the README.md thats how most of the projects are done. The pros of this is that README.md get version controlled and can 'be updated with every release.

**Comment by pilotnbr1** [comment]
@bortek when clicking on the “home” link from the insights traffic page, it goes here
https://github.com/bortek/EZ-WifiBroadcast/wiki

So interestingly enough the readme.md page which is 
https://github.com/bortek/EZ-WifiBroadcast is the number 2 page with unique visitors

The analytics don’t make it very clear how they are getting there. Google being the number 1 source of traffic perhaps points to the “home” page as most relevant?!

Both pages could definitely be improved but I suggest most “advertising” go to the page attracting the most unique visitors- https://github.com/bortek/EZ-WifiBroadcast/wiki . This page currently does a poor job introducing wbc and is cluttered imho.

@careyer yes good idea with a YouTube channel!

**Comment by richardbmx** [comment]
@pilotnbr1  Hi, my name is Richard. I'm a graphic designer and I love making contributions for open source projects. I see that they want a new logo for your application. If you wish, I can do it for you without any problem ... I can present some modern ideas that would improve the appearance and image of your project

**Comment by htcohio** [comment]
@richardbmx,

My brother-in-law is working on a couple logos in his free time as we speak.  They just had a baby a month ago so his hands are kind of full but he said he'd have a few more options over to me this week.  I'll post a couple of the samples on rcgroups thread for you to take a look at but, it would be great if a couple people could come up with ideas.

**Comment by htcohio** [comment]
@pilotnbr1 

Hey, as you may have previously seen I have been planning on and will be building on my existing Channel for YouTube over the next couple of weeks.  I've been putting together the studio in my shop and picked up a couple camcorders and gear that will make the videos relatively high quality.

My plan is to have organized instructional playlists for basic builds and setups in addition to some other fpv related topics.

It will be primarily centered around this fpv system but I also plan on product reviews and comparisons.  Feel free to use any of them on a future page.  Most are of not the best quality but they do have some good content and information related to the basic setup.

The first video I have planned will be something to the effect of "wifibroadcast for dummies" it will be a brief intro with credits and how to get your first system up and running quickly.

My Hope Is that all of the initial setup could be completed through one of the Android apps eliminating the need to alter text files at all.  I'm not sure how far out that part is yet but I'd like to make make sure that the initial content is based on the most likely integration of changing the settings.

Antennas also seem to be a really hot topic.  I've discovered several Surprises with supposedly good antennas with a vector impedance analyzer that I picked up recently.  I'm hoping to shed some insight on that to help others.  The best Solutions actually seem to be handmade antennas such @Khomyakk's.  

I'm completely open to ideas as to what you guys think should be prioritized from the video standpoint.  

Hopefully more to follow very soon.  )



My channel:
https://www.youtube.com/channel/UC-iXe0UW2e9UGXkBAD8CBRQ

**Comment by bortek** [comment]
@richardbmx Yes please do so, come up with some suggestion. We can can then vote and select whichever we thinks suits best to the project. Thank!

**Comment by careyer** [comment]
@richardbmx, thank you very much for offering your help. I can't wait with what you will come up with. :-)

Von meinem iPhone gesendet

**Comment by RespawnDespair** [comment]
@richardbmx , i have scrolled through your portfolio, some great logo designs there! @htcohio 's brother-in-law is also going to provide some designs, but i'm very interested in what you would come up with.

When we progress further we would also need some illustrations done for the guides and howto's. I would love to have one consistent visual style for those as well. Perhaps we could look into that when the basics are done in regards to the 1.6 generated image.

**Comment by bortek** [comment]
If you google for "Ez wifibroadcast" the first link is to Wiki which has Home in the title (perhaps that why first) and the second link is to landing page. But most people look at the landing page.  I can later try to use the sitemap tool to alter the search to make the landing page show first in the google search results. 

But first things first. We need to fill in with more info and images, graphs so it has a visual appeal. 

What do you think about this proposal? Feel free to comment and come up with suggestion.

Landing page (README.md)
- What this project is about
- Description with images
- Download link to the latest image
- Links to the Wiki for more info
- Licensing, disclaimer 
- Code of conduct 

Wiki first page 
- How to setup guide/getting started

**Comment by RespawnDespair** [comment]
@bortek looks good to me... I once wanted to make a 'real' manual as i've seen in other projects. So i created and open-source gitbook account.
[https://ez-wifibroadcast.gitbook.io/project/](https://ez-wifibroadcast.gitbook.io/project/)

I think we should stick to the wiki for now, but at least the url is claimed within gitbook 😄

**Comment by careyer** [comment]
@RespawnDespair / @bortek : Github stuff answered my question from yesterday! It is simply not possible to set the Wiki Mainpage as the default landing page of your repository. However they advised to take a look at https://pages.github.com - this seems to be very similar to what @RespawnDespair has suggested.

**Comment by careyer** [comment]
> @richardbmx , i have scrolled through your portfolio, some great logo designs there! @htcohio 's brother-in-law is also going to provide some designs, but i'm very interested in what you would come up with.

@richardbmx I did so as well... and I really like the light and modern style of the logos I see there! Awesome work! I'm excited.

>When we progress further we would also need some illustrations done for the guides and howto's. I >would love to have one consistent visual style for those as well.

+1 on this one as well.. I very much enjoy for example the very consistent visual appeal of the Ardupilot manual http://ardupilot.org/copter/docs/frame-type-configuration.html / I really enjoy browsing those sites.

**Comment by pilotnbr1** [comment]
@richardbmx @htcohio Yes more logo options would be better than less! Personally I had something in mind, that captures the essence of the project- like a camera lenses, WiFi symbol, quadcopter, etc. But open minded and curious to see what is presented.

It is a little strange to go through this wiki and not see any demo videos, pics of osd, or much multimedia- funny for a project that is about hd video, quality, and visual appeal lol

@bortek yes your proposal sounds good. That is logical and seems in line with standard operating practice on github.

All good ideas here!

**Comment by pilotnbr1** [comment]
@bortek I have one addition to the readme which is license info, recognition of contributed/borrowed code, and perhaps some sort of disclaimer for legal. This will also make us more compliant with the rest of github.

Additionally github likes to see some sort of code of conduct. I just suggest some boiler plate standard template but most other large projects have this and seems to be good practice.

**Comment by bortek** [comment]
Understood about the limitations of landing page. Let's not split attention too much for now, we can take a look at alternative pages on a later stage.

I knew a question about licensing would sooner or later would pop up, so here ot came 😁.  I have an idea and will add it to Readme.

**Comment by htcohio** [comment]
Yeah, @richardbmx if you were to handle the logo I think it would be
better.  we would have more control over the graphics and the turnaround
time would probably be quicker for any add-ons or future changes.

Portfolio looks awesome!

**Comment by richardbmx** [comment]
perfect I will start working on this

**Comment by careyer** [comment]
@richardbmx : you might want to look into this new thread: #164

**Comment by richardbmx** [comment]
I would like to present my proposal based on all your ideas I have made this logo that expresses the camera and the Wi-Fi connection very well in addition to this it can be adapted to any background color and size without losing quality. If you want any change you should only ask for it.

![logo115](https://user-images.githubusercontent.com/35604837/47197761-3541ce80-d336-11e8-8e18-c27f28773c36.png)

**Comment by bortek** [comment]
I have update the landing page text. Can someone take a look and give me a feedback on the text content, don't be shy giving the critic. 

https://github.com/bortek/EZ-WifiBroadcast/blob/develop/README.md

**Comment by pilotnbr1** [comment]
@bortek fixed a couple typos. FYI “infamous” has negative connotations (famous in a bad way) don’t think that’s what you meant with Befinitiv:) I have huge respect with you guys writing this in a 2nd language and I barely write English like it’s my first lol

I am also not sure about describing wbc as a unidirectional link. With rc and some telemetry wbc is bidirectional- so maybe not describing the capabilities of wbc completely in that regard. Otherwise really liked it! Big improvement over the previous version! Thanks for the hard work!

**Comment by bortek** [comment]
good findings, I have changed uni to bi and in*famous have been already corrected by you. Will continue to update it in the coming days.

**Comment by careyer** [comment]
added a missing "based **on**" :-) Otherwise pretty much perfect!

**Comment by careyer** [comment]
@bortek : You you confirm that you have received the 31km flight video on your NextCloud?

**Comment by bortek** [comment]
Oh yeah I can see 117 files in your folder. That took it's time. Will fetch it now down to my PC then what we can do with it :)

**Comment by bortek** [comment]
@ALL 
I've added short list of main features to the landing page. Let me know if I missing something important or just edit it yourself.

Now I would like to have one or two images on the landing page e.g. describing the setup. I can draw some nice diagram and put it up there. But I need ideas on what should it be. Preferably it should be something different from what DroneBridge has on it's landing page. Please come with any ideas you have on your mind and we can shape it down to something workable.

**Comment by careyer** [comment]
Great! You might want to add RC control as well (Mavlink, SUMD, UBUS, …)

**Comment by rodizio1** [comment]
Thanks a lot for your help guys.
